content/blog/getting-started-with-spark-on-mapr-sandbox.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Getting Started with Spark on MapR Sandbox
+date: 2021-10-14T06:01:00.000Z
+author: MapR Tutorials
+authorimage: /img/Avatar1.svg
+---
+
+
+**Editor’s Note:** MapR products and solutions sold prior to the acquisition of such assets by Hewlett Packard Enterprise Company in 2019 may have older product names and model numbers that differ from current solutions. For information about current offerings, which are now part of HPE Ezmeral Data Fabric, please visit <https://www.hpe.com/us/en/software/ezmeral-data-fabric.html>
+
+At MapR, we distribute and support Apache Spark as part of the MapR Converged Data Platform. This tutorial will help you get started with running Spark applications on the MapR Sandbox.
+
+## Prerequisites
+
+**HARDWARE REQUIREMENTS**
+
+* 8GB RAM, multi-core CPU
+* 20GB minimum HDD space
+* Internet access
+
+**SOFTWARE REQUIREMENTS**
+
+* A hypervisor. This example uses VMware Fusion 6.0.2 on OSX; however, other VMware products could be used instead. Additionally, [VirtualBox](https://www.virtualbox.org/) can be used
+* A virtual machine image for the MapR Sandbox. Spark comes pre-loaded with version 5.0 and later of the MapR Sandbox.
+
+**Starting up and Logging into the Sandbox**\
+Install and fire up the Sandbox using the instructions here: <https://docs.datafabric.hpe.com/62/MapRContainerDevelopers/RunMapRContainerDevelopers.html>. Once you are able to login to the web interface for the Sandbox, you are ready to start setting up Spark.
```

Comment:
Suggest you make the URL a hotlink. 
Install and fire up the Sandbox using the instructions [here](https://docs.datafabric.hpe.com/62/MapRContainerDevelopers/RunMapRContainerDevelopers.html)

Reply:
done